packages/core/src/components/popover/popover.tsx

```diff
@@ -271,16 +281,21 @@ export class Popover extends AbstractComponent<IPopoverProps, IPopoverState> {
         }, className);
         targetProps.ref = this.refHandlers.target;
 
+        const childrenBaseProps = this.props.openOnTargetFocus && this.isHoverInteractionKind()
+            ? { tabIndex: 0 }
```

Comment:
ehh I'm not so sure... #724 could take a while and would be a large break so it might be a while before end users see this feature in practice